chatbot-ui/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
This README should not be boiler-plate, it will contain a description of how to run our SalesGPT UI.